playwright/src/test/java/com/microsoft/playwright/impl/ListenerCollectionTest.java

```diff
@@ -0,0 +1,15 @@
+package com.microsoft.playwright.impl;
+
+import org.junit.jupiter.api.Test;
+
+class ListenerCollectionTest {
+
+  @Test
+  void shouldHandleNullListener() {
```

Comment:
We prefer integration tests in the project, can you convert it in a test that uses Playwright public API?

Reply:
sure, changed